envoy/cilium/accesslog.proto

```diff
@@ -20,11 +20,30 @@ enum EntryType {
 }
 
 message HttpLogEntry {
+  Protocol http_protocol = 1;
+
+  // Request info that is also retained for the response
+  string scheme = 2;      // Envoy "x-forwarded-proto", e.g., "http", "https"
+  string host = 3;        // Envoy ":authority" header
+  string path = 4;        // Envoy ":path" header
+  string method = 5;      // Envoy ":method" header
+
+  // Request headers not included above
+  repeated KeyValue headers = 6;
+
+  // Response info
+  uint32 status = 7;      // Envoy ":status" header, zero for request
+}
+
+// Deprecated old message format, to be removed in a future Cilium release
```

Comment:
I wouldn't say that this is deprecated. :)
I would only say that some of the fields documented as deprecated will be removed in a future Cilium release.

Reply:
The comment was a remnant of an earlier work-in-progress version, thanks for spotting it!